nextcord/guild.py

```diff
@@ -1682,6 +1694,9 @@ async def edit(
             The new channel that is used for public updates from Discord. This is only available to
             guilds that contain ``PUBLIC`` in :attr:`Guild.features`. Could be ``None`` for no
             public updates channel.
+        invites_disabled: :class:`bool`
+            Whether the invites should be paused for the guild.
+            This will prevent new users from joining said guild.
```

Comment:
```suggestion
        invites_disabled: :class:`bool`
            Whether the invites should be paused for the guild.
            This will prevent new users from joining said guild.
            
            .. versionadded:: 2.4
```